cortex-python-lib-examples/gcs-connection-skill/docs/gcs-connection-skill/README.md

```diff
@@ -0,0 +1,90 @@
+  "activationId": "4b6ce400-5c34-4cb4-8cfd-2ada7be09c93"
+}
+```
+
+You can alternatively run `make all` to run steps 1 - 5 in sequence. With the `activationId` you can check for the status of the job by running `cortex agents get-activation <activationId>`. The skill invocation should result in a status of `COMPLETE` with the message `"Connection: gcs-german-credit read successfully"` being the Job response (logs).
+```
+$ cortex agents get-activation 4b6ce400-5c34-4cb4-8cfd-2fab3be09c93
+{
+  "success": true,
+  "requestId": "4b6ce400-5c34-4cb4-8cfd-2fab3be09c93",
+  "skillName": "gcs-connection-skill",
+  "inputName": "params",
+  "projectId": "test-project",
+  "username": "user@example.com",
+  "payload": {
+    "connection_name": "gcs-german-credit"
+  },
+  "sessionId": "4b6ce400-5c34-4cb4-8cfd-2ada7be09c93",
+  "start": 1651517957034,
+  "status": "COMPLETE",
+  "end": 1651517966356,
+  "response": "2022-05-02T18:59:19Z scuttle: Scuttle 1.3.6 starting up, pid 1\n2022-05-02T18:59:19Z scuttle: Logging is now enabled\n2022-05-02T18:59:19Z scuttle: Blocking until Envoy starts\n2022-05-02T18:59:19Z scuttle: Polling Envoy (1), error: internal_service: dial tcp 127.0.0.1:15000: connect: connection refused\n2022-05-02T18:59:19Z scuttle: Blocking finished, Envoy has started\nConnection: gcs-german-credit read successfully\n2022-05-02T18:59:20Z scuttle: Kill received: (Action: Stopping Istio with API, Reason: ISTIO_QUIT_API is set, Exit Code: 0)\n2022-05-02T18:59:20Z scuttle: Stopping Istio using Istio API 'http://localhost:15020' (intended for Istio >v1.2)\n2022-05-02T18:59:20Z scuttle: Received signal 'child exited', passing to child\n2022-05-02T18:59:20Z scuttle: Sent quitquitquit to Istio, status code: 200\n"
+}
+```
+
+Or by finding the logs from the corresponding Task (check `cortex tasks list` to find the entry with the matching `activationId`), e.g.
+```
+$ cortex tasks logs test-project-gcs-connection-skill-gcs-connection-rs2g2p
+2022-05-02T18:59:19Z scuttle: Scuttle 1.3.6 starting up, pid 1
+2022-05-02T18:59:19Z scuttle: Logging is now enabled
+2022-05-02T18:59:19Z scuttle: Blocking until Envoy starts
+2022-05-02T18:59:19Z scuttle: Polling Envoy (1), error: internal_service: dial tcp 127.0.0.1:15000: connect: connection refused
+2022-05-02T18:59:19Z scuttle: Blocking finished, Envoy has started
+Connection: gcs-german-credit read successfully
+2022-05-02T18:59:20Z scuttle: Kill received: (Action: Stopping Istio with API, Reason: ISTIO_QUIT_API is set, Exit Code: 0)
+2022-05-02T18:59:20Z scuttle: Stopping Istio using Istio API 'http://localhost:15020' (intended for Istio >v1.2)
+2022-05-02T18:59:20Z scuttle: Received signal 'child exited', passing to child
+2022-05-02T18:59:20Z scuttle: Sent quitquitquit to Istio, status code: 200
+```
+
+For more details about how to build skills go to [Cortex Fabric Documentation - Development - Develop Skills](https://cognitivescale.github.io/cortex-fabric/docs/development/define-skills)
```

Comment:
this link looks old

Reply:
Yup, definitely is. I copied that from a different example, so I've included those updates here.